src/enums.h

```diff
@@ -164,6 +164,12 @@ enum phase_id {
     PNULL, SOLID, LIQUID, GAS, PLASMA
 };
 
+enum verbal_shout_id : int {
```

Comment:
Would fit better in sounds.h if it would be visible to everything relevant from there.